disasterinfosite/templates/user_interactions.html

```diff
@@ -0,0 +1,170 @@
+<div role="content-info" id="user-interaction-container">
+  {% if username and profile %}
+    <div id="user-button-container--logged-in" class="row vert-pad-medium">
+      <div class="small-12 medium-6 columns">
+        <p><strong>Welcome back!</strong></p>
+        <p>If your username isn't {{ username }}, you might want to log out.</p>
+      </div>
+      <div class="small-12 medium-6 columns user-interaction__buttons">
+        {% csrf_token %}
+        <a href="#user-login-container" class="button radius caps button--logout">Log Out</a>
+        <a href="#user-info-container" class="button radius caps button--update">Update My Info</a>
+      </div>
+    </div>
+    <div id="user-button-container" class="row vert-pad-medium hide">
+  {% else %}
+    <div id="user-button-container" class="row vert-pad-medium">
+  {% endif %}
+      <div class="small-12 medium-6 columns">
+        <p><strong>Why should you sign up?</strong></p>
+        <p>Sign up for free to access customized hazard preparedness information. By providing your address, you will get updates about seasonal hazards and ongoing incidents specific to your location. We want you be better prepared so you can keep your family, friends, and neighbors safe when a disaster strikes! Personal information will be protected and secure.</p>
+      </div>
+      <div class="small-12 medium-6 columns user-interaction__buttons">
+        <a href="#user-login-container" class="button radius caps button--login">Log In</a>
+        <a href="#user-signup-container" class="button radius caps button--signup">Sign Up</a>
+      </div>
+    </div>
+  <div id="user-login-container" class="row vert-pad-medium">
+    <form id="user-login__form">
+      {% csrf_token %}
+      <div class="small-12 medium-6 columns">
+        <label>Username (your email address)
+          <input type="email" id="user-login__username" required placeholder="Required."/>
```

Comment:
Currently it is possible for admin users' usernames to not be email addresses.  We should either change that or loosen the validation here.